modules/core/src/parallel.cpp

```diff
@@ -816,9 +897,10 @@ int cv::getNumberOfCPUs(void)
             numCPU = 1;
     }
 
-    return (int)numCPU;
+    unsigned ncpus = min_non_zero(concurentThreadsSupported, (unsigned)numCPU);
+    return ncpus != 0 ? ncpus : 1;
 #else
-    return 1;
+    return concurentThreadsSupported != 0 ? concurentThreadsSupported : 1;
```

Comment:
> return ncpus != 0 ? ncpus : 1;
> return ncpus != 0 ? ncpus : 1;
> return ncpus != 0 ? ncpus : 1;
> return ncpus != 0 ? ncpus : 1;
> return concurentThreadsSupported != 0 ? concurentThreadsSupported : 1;

There are 5 similar returns above.
Please make return once.
Rename `concurentThreadsSupported` => `ncpus` to reduce code.

Reply:
Oh ok, thanks, will do